data.rkt

```diff
@@ -753,7 +784,49 @@
          #:host (host host-id-manchester-makerspace)
          #:registration-url (u "")
          #:talks
-         '())
+         (list
+           (T #:speaker (speaker speaker-id-tigran-khandkar)
+              #:title "Item and Effect Creation in Minecraft"
+              #:description "Custom super-powerful items in Minecraft using command scripts and scoreboards."
+              #:artifacts '()
+              #:website #f
+              #:references '())
+
+           (T #:speaker (speaker speaker-id-kevin-kadow)
+              #:title "Manchester Makerspace's Custom Doorbell"
+              #:description "A networked and remote doorbell and surveillance system for the Manchester Makerspace."
+              #:artifacts '()
+              #:website #f
+              #:references '())
```

Comment:
Am I remembering correctly that this was built with ESP32? This maybe a good ref to add: https://en.wikipedia.org/wiki/ESP32

Hoping Kevin will chime in with more!